src/Migrations/LegacyBackupDirAwareInterface.php

```diff
@@ -4,6 +4,6 @@
 interface LegacyBackupDirAwareInterface
 {
     //backup_dir parameter is not used since 2018 versions, but it is necessary to migrate from older versions
-    public function setLegacyBackupDir(string $backupDir): void;
```

Comment:
Is this type removal really necessary?

Reply:
No, my bad. I did not want to commit this.